app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_in = command.split()[1]
```

Comment:
use command.split() only one time, and you must check that command is `cp` and there are exactly 3 arguments